boom! testing cake-0.1.0 against stable for nightly
boom! running: cargo +stable build --frozen
boom! creating container for: cargo +stable build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable build --frozen cargobomb`
blam! 4286803a702b1764b3754a3493483c4a01fb7f81cc521d37a88db92d1b029b66
boom! running `docker start -a 4286803a702b1764b3754a3493483c4a01fb7f81cc521d37a88db92d1b029b66`
kablam!    Compiling cake v0.1.0 (file:///source)
kablam!     Finished debug [unoptimized + debuginfo] target(s) in 0.11 secs
boom! running `docker rm -f 4286803a702b1764b3754a3493483c4a01fb7f81cc521d37a88db92d1b029b66`
blam! 4286803a702b1764b3754a3493483c4a01fb7f81cc521d37a88db92d1b029b66
boom! running: cargo +stable test --frozen --no-run
boom! creating container for: cargo +stable test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable test --frozen --no-run cargobomb`
blam! 19454e31c4dbe114aad3a4fa975a645c813052e1d054447bac38b1e218d9ea02
boom! running `docker start -a 19454e31c4dbe114aad3a4fa975a645c813052e1d054447bac38b1e218d9ea02`
kablam!    Compiling cake v0.1.0 (file:///source)
kablam!     Finished debug [unoptimized + debuginfo] target(s) in 1.47 secs
boom! running `docker rm -f 19454e31c4dbe114aad3a4fa975a645c813052e1d054447bac38b1e218d9ea02`
blam! 19454e31c4dbe114aad3a4fa975a645c813052e1d054447bac38b1e218d9ea02
boom! running: cargo +stable test --frozen
boom! creating container for: cargo +stable test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable test --frozen cargobomb`
blam! d6367b97b4638f7397477c931e39dd399d92feca7a49b243f83c0713926d286c
boom! running `docker start -a d6367b97b4638f7397477c931e39dd399d92feca7a49b243f83c0713926d286c`
kablam!     Finished debug [unoptimized + debuginfo] target(s) in 0.0 secs
kablam!      Running /target/debug/deps/cake-c1a0297a1a3209d2
blam! 
blam! running 0 tests
blam! 
blam! test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured
blam! 
kablam!    Doc-tests cake
blam! 
blam! running 4 tests
blam! test cmd_0 ... FAILED
blam! test cmd_1 ... FAILED
blam! test cmd_2 ... FAILED
blam! test build_0 ... FAILED
blam! 
blam! failures:
blam! 
blam! ---- cmd_0 stdout ----
blam! 	error: macro undefined: 'cmd!'
blam!  --> <anon>:2:1
blam!   |
blam! 2 | cmd!("ls", "/"); // 'ls /'
blam!   | ^^^
blam! 
blam! error: aborting due to previous error
blam! 
blam! thread 'cmd_0' panicked at 'Box<Any>', ../src/librustc_errors/lib.rs:461
blam! thread 'cmd_0' panicked at 'couldn't compile the test', ../src/librustdoc/test.rs:275
blam! 
blam! ---- cmd_1 stdout ----
blam! 	error: macro undefined: 'cmd!'
blam!  --> <anon>:2:1
blam!   |
blam! 2 | cmd!("ls", "."; in "/");
blam!   | ^^^
blam! 
blam! error: aborting due to previous error
blam! 
blam! thread 'cmd_1' panicked at 'Box<Any>', ../src/librustc_errors/lib.rs:461
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! thread 'cmd_1' panicked at 'couldn't compile the test', ../src/librustdoc/test.rs:275
blam! 
blam! ---- cmd_2 stdout ----
blam! 	error: macro undefined: 'cmd!'
blam!  --> <anon>:2:1
blam!   |
blam! 2 | cmd!("ls", "."; where BLAH = "/"; where BLUH = "!");
blam!   | ^^^
blam! 
blam! error: aborting due to previous error
blam! 
blam! thread 'cmd_2' panicked at 'Box<Any>', ../src/librustc_errors/lib.rs:461
blam! thread 'cmd_2' panicked at 'couldn't compile the test', ../src/librustdoc/test.rs:275
blam! 
blam! ---- build_0 stdout ----
blam! 	error[E0468]: an `extern crate` loading macros must be at the crate root
blam!  --> <anon>:3:1
blam!   |
blam! 3 | extern crate cake;
blam!   | ^^^^^^^^^^^^^^^^^^
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'build_0' panicked at 'Box<Any>', ../src/librustc/session/mod.rs:191
blam! 
blam! 
blam! failures:
blam!     build_0
blam!     cmd_0
blam!     cmd_1
blam!     cmd_2
blam! 
blam! test result: FAILED. 0 passed; 4 failed; 0 ignored; 0 measured
blam! 
kablam! error: test failed
boom! running `docker rm -f d6367b97b4638f7397477c931e39dd399d92feca7a49b243f83c0713926d286c`
blam! d6367b97b4638f7397477c931e39dd399d92feca7a49b243f83c0713926d286c
